web/src/inspector.js

```diff
@@ -222,13 +274,18 @@ export class Inspector extends EventTarget {
       if (Inspector._needsExtraInfo(torrents)) {
         fields.push(...Torrent.Fields.InfoExtra);
       }
+      if (this.settings_dirty) {
+        this.settings_dirty = false;
+        fields.push(...Torrent.Fields.SettingsExtra);
+      }
 
       controller.updateTorrents(ids, fields);
     }
   }
 
   _updateCurrentPage() {
     const { elements } = this;
+    this.updating = true;
```

Comment:
Unlike with other fields in the torrent info, editing these has side effects. Updates coming from the remote side or from simply rearranging the page as a user flips through tabs shouldn't push settings changes to the remote, this mechanism disables that while an update is in progress.